thewalrus/tests/test_quantum.py

```diff
@@ -1355,3 +1356,16 @@ def test_find_classical_subsystem_thermal(hbar):
     cov = O @ cov @ O.T
     k = find_classical_subsystem(cov, hbar=hbar)
     assert k == nmodes
+
+
+def test_tvd_cutoff_bound():

```

Comment:
This is a great test. My only suggestion would be to use `parametrize` to perform an independent test for each value of the cutoff, i.e., use the decorator `@pytest.mark.parametrize("cutoff", range(10))` and write the test for a single value of the cutoff, which then gets iterated over all values in `range(10)`

Reply:
it is already checking all the cutoffs up to 10.

Reply:
The function returns the bounds in tvd for all the cutoffs between 0 and the cutoff given by the user.